Wiki-style-guide.md

```diff
@@ -0,0 +1,13 @@
+## Text markup
+
+* Use markdown headers since we can create links to headers in the wiki. This is not possible with long lists or "headers" that are just bold.
+* Use quotes (`""`) for single-line quotes. This includes button names.
+* Use markdown block quotes for large quotations.
+* Use inline code and code block markup (backticks in markdown) only for code, where the text needs to be monospaced (e.g. ASCII art), or where the user needs to be able to distinguish individual characters (e.g. exploration IDs). Don't use backticks when quotes will do.
+
+## Writing style
+
+* Use sentence case for headings instead of title case.
```

Comment:
Link to a definition of sentence case / title case, or give examples.

In general, explain jargon.

Reply:
done